wizelineacademy/react-native-workshop
# Feat - Add ESLint with Airbnb

#### What does this PR do?

Adds the Airbnb ESLint standard.

#### Where should the reviewer start?

In the `.eslintrc` file.

#### How should this be manually tested?
1. Install the ESLint dependencies.
    ```bash
    yarn
    ```
    **Note:** The required versions are Node.js >= 8.11.3 and Yarn >= 1.5.1. 
2. Run the lint script.
    ```bash
    yarn lint
    ```
3. Verify that there are no lint errors.

#### Any background context you want to provide?



I added the rules from the [`eslint-plugin-react-native`](https://github.com/Intellicode/eslint-plugin-react-native).

#### What are the relevant tickets?



N/A

#### Checklist
